frontend/src/toolbar/flags/featureFlagsLogic.ts

```diff
@@ -0,0 +1,77 @@
+import { toolbarLogic } from '~/toolbar/toolbarLogic'
+
+export const featureFlagsLogic = kea<featureFlagsLogicType>({
+    actions: {
+        getUserFlags: true,
+        setOverriddenUserFlag: (flagId: number, overrideValue: string | boolean) => ({ flagId, overrideValue }),
+        deleteOverriddenUserFlag: (overrideId: number) => ({ overrideId }),
+    },
+
+    loaders: ({ values }) => ({
+        userFlags: [
+            [] as CombinedFeatureFlagAndOverride[],
+            {
+                getUserFlags: async (_, breakpoint) => {
+                    const response = await toolbarFetch('api/feature_flag/my_flags')
+                    breakpoint()
+                    if (response.status === 403) {
+                        return []
+                    }
+                    const results = await response.json()
+                    return results
+                },
+                setOverriddenUserFlag: async ({ flagId, overrideValue }, breakpoint) => {
+                    const response = await toolbarFetch(
+                        'api/projects/@current/feature_flag_overrides/my_overrides',
+                        'POST',
+                        {
+                            feature_flag: flagId,
+                            override_value: overrideValue,
+                        }
+                    )
+                    breakpoint()
+                    if (response.status === 403) {
+                        return []
+                    }
+                    const results = await response.json()
+                    ;(window['posthog'] as PostHog).featureFlags.reloadFeatureFlags()
+
+                    return [...values.userFlags].map((userFlag) =>
+                        userFlag.feature_flag.id === results.feature_flag
+                            ? { ...userFlag, override: results }
+                            : userFlag
+                    )
+                },
+                deleteOverriddenUserFlag: async ({ overrideId }, breakpoint) => {
+                    const response = await toolbarFetch(
+                        `api/projects/@current/feature_flag_overrides/${overrideId}`,
+                        'DELETE'
+                    )
+                    breakpoint()
+                    if (response.status === 403) {
+                        return []
+                    }
+                    ;(window['posthog'] as PostHog).featureFlags.reloadFeatureFlags()
+
+                    return [...values.userFlags].map((userFlag) =>
+                        userFlag?.override?.id === overrideId ? { ...userFlag, override: null } : userFlag
+                    )
+                },
+            },
+        ],
+    }),
+
+    events: ({ actions }) => ({
+        afterMount: () => {
+            actions.getUserFlags()
+            ;(window['posthog'] as PostHog).onFeatureFlags((_, variants) => {
+                toolbarLogic.actions.updateFeatureFlags(variants)
+            })
```

Comment:
So this works when you're running the toolbar on the posthog app itself, but as soon as you're running the toolbar on someone else's website, we'd override the toolbar's feature flags (flags such as "are feature flags enabled in the toolbar?") with the site's feature flags ("show hero banner 2 on kea.js.org"). Fine for development for quick feedback, but should eventually be removed.